.github/workflows/auto-approver.yaml

```
@@ -0,0 +1,25 @@
+name: auto-approver
+run-name: CI for approving PRs
+
+on:
+  push:
+    branches:
+      - "renovate/**"
+
+jobs:
+  autoapprove:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
```

Can we update to v4 like in other repos?
```suggestion
      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```